src/_data/site.yml

```diff
@@ -96,5 +96,11 @@ home:
       position: 🚀 SysAdmin
       username: rjm
       image: /assets/img/committee/ruan.webp
+  sponsors:
+    - name: Rewriting the Code
+      tier: Sponsor
```

Comment:
If we remove it we have to remove it for all sponsors, including the primary sponsor.

Reply:
I should probably clarify why that happens.
If we had 'tier' only for the primary sponsors, then it would error out for every non-primary sponsor as 11ty would be looking for `sponsor.tier` which only exists in Primary sponsors.
Theoretically, I could keep tier on everything, and have it as 'blank'. Then an if check checks if it's == blank, and if so, it won't render.

I will try later.

Reply:
Removed Sponsor tier text.
Making it an if breaks the rendering.